.swiftlint.yml

```diff
@@ -1,9 +1,8 @@
 disabled_rules:
   - line_length
   - type_name
-  - valid_docs
   - function_body_length
-  - variable_name
+  - identifier_name
```

Comment:
> Loading configuration from '.swiftlint.yml'
__'variable_name'__ rule has been renamed to __'identifier_name'__ and will be completely removed in a future release.
configuration error: 'valid_docs' is not a valid rule identifier